### PR TITLE
About screen

### PR DESCRIPTION
**Pull request by kistitan**
- Adding version and feature history to a new About screen accessible through the menu
- Display snackbar notification in case of first start after new version

### CODE REVIEW

**Comment by jacortinas** [comment]
👍 lgtm

File: FetLife/fetlife/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -60,5 +58,49 @@
...
+                        <li>Read and respond to Conversations</li>
+                        <li>View your list of Friends</li>
+                        <li>Start new Conversation with a Friend</li>
+                        <li>Receive a push notification when you receive a new Message</li>
+                    </ul>
+
+            <p/>
+        ]]>
+    </string>
+    <string name="snakckbar_version_notification">Check out what\'s new in v%1$s at \'About\' screen</string>
```
Comment (philippt) [inline FetLife/fetlife/src/main/res/values/strings.xml]: one `k` too much in snakckbar_version_notification?

**Comment by philippt** [comment]
:+1: lgtm2

File: FetLife/fetlife/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -60,5 +58,49 @@
...
+
+            <p/>v1.3<br/>
+                    <ul>
+                        <li>Receive broader set of Notification</li>
+                        <li>View your Notification History</li>
+                        <li>Get information about new features at about screen</li>
+                        <li>Use hardware menu button for opening the menu</li>
+                    </ul>
+
+                </font>
```
Comment (reneklacan) [inline FetLife/fetlife/src/main/res/values/strings.xml]: where does font tag starts? weird xml

Comment (kistitan) [inline FetLife/fetlife/src/main/res/values/strings.xml]: Nowehere haha, I remove the end tag.

Yep it's an xml containing some html string.
It is used sometimes due to lower maintenance cost (aka lazyness).

Other option would be to keep all the lines in separate xml entry. 
But that also not very nice.

**Comment by Daddy** [comment]
Lgtm

**Pull request merged**